docs/details/logs.md

```diff
@@ -20,25 +23,18 @@ rolls over.
 
 ##### macOS Unified Logging System (ULS)
 
-Currently all of the most recent releases of Santa are built with the macOS
-10.11 SDK. This allows Santa to continue to log to Apple System Logger (ASL)
-instead of ULS. However, on macOS 10.12+ all of the Kernel logs are sent to ULS.
-See the KEXT Logging section below for more details.
+As Santa has been built with macOS 10.12+ SDKs for several releases, Santa's logs
+are also sent to ULS.
 
-If you are building Santa yourself and using the macOS 10.12+ SDKs, Santa's logs
-will be sent to ULS.
+Leveraging this capability, `show` can be used to view all santa-specific logs in
```

Comment:
s/santa/Santa

Reply:
I futzed around a bit to try to be accurate and readable in a separate commit, 3e7b01d, PTAL

Reply:
LGTM